www/templates/main-recommendations.html

```diff
@@ -8,22 +7,23 @@
      <link rel="stylesheet" href="css/main-recommend.css">
      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
      <link rel="stylesheet" type="text/css" href="slick/slick.css"/>
-  <ion-content class="has-header" overflow-scroll="true" padding="false" ng-controller="RecommendationsCtrl">
+  <ion-content class="has-header" overflow-scroll="true" padding="false">
     <div id = "scrolling">
      <center><div class="recommendbox" id = "0">
-       <div class = "larger_font"> {{recs.type}} </div>
+       <div class = "larger_font"> Cafes and Beverage </div>
        <h4>
-       <p> {{recs.name}} </p>
-         <p> {{recs.address}} </p>
-           <p> {{getDistance(recs.coord[0], recs.coord[1])}}</p>
-           <img src="img/small/small_{{recs.stars}}.png" widsth="75" />
+
+       <p> Ernie's Espresso </p>
+         <p>1234 Sesame St. </p>
+           <p> 0.2 Miles Away </p>
+           <img src="img/small/small_4.png" width="75" />
```

Comment:
I don't see how the suggestion will show up correctly from Vanessa. You just removed all the template code. How will you format the received suggestion?